### DISCUSSION TITLE
IMDB import

### DISCUSSION THREAD

**Discussion by christaikobo** [sbondCo/Watcharr#720] [Q&A] [answered]
Can someone please tell me how to import IMDB export? I saw in an pr that it was supported alongside TMDB import, but it doesn't work for me. I clicked the TMDB import button and tries to import my IMDB export (csv) and it just doesn't work. 

Thanks!

**Comment by IRHM** [comment, marked as answer]
Hi @terrytw, I think I found the issue that seems to portray that imdb importing support was added (#215?), I think I likely added tmdb support and totally forgot about imdb.

I will create an issue to track this feature.

**Reply by christaikobo** [reply]
Thanks for the confirmation. Yes it is the issue you linked, the title says IMDB/TMDB import so I was trying IMDB import.
